docker/Dockerfile.pybot

```diff
@@ -0,0 +1,15 @@
+FROM python:3.10-slim
+
+ARG BOT_NAME
+ARG BOT_PORT
+
+WORKDIR /app
+COPY ./ ./
+COPY ./examples/randbot.py ./main.py
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+EXPOSE 50051
```

Comment:
I'd add the game server address as an argument too, so that we can set a different game server address if needed.